api/globalConfig.py

```diff
@@ -54,6 +55,7 @@ def __init__(self):
         self.dbNameSearch = config.get('DB-Section', 'name-search', fallback='')
         self.dbUser = config.get('DB-Section', 'user', fallback='')
         self.dbPass = config.get('Secrets-Section', 'mysql-password', fallback='')
+        self.sqlalchemy = SQLAlchemy(engine_options={'future': True})
```

Comment:
this line needs to be added in load_config method as well. if some person in admin ui will change sql variables they have to change in flask app as well.